### PR TITLE
refacto(layer): deprecate adding a layer with only a configuration 

### PR DESCRIPTION
**Pull request by zarov**
As suggested in #880 I'm starting to lock the "finale API" of iTowns, with retrocompatibility of course.

First commit is the locking, with all examples changed as needed.

Second commit is a new tutorial. This could be done outside this PR, but I wrote it alongside these modifications, so I would like to add it together.
This tutorial will be modified further along the locking of the API.

### CODE REVIEW

**Comment by zarov** [comment]
This PR also relies on #904 being merged.

File: examples/globe_travel.html
Comment (gchoqueux) [inline examples/globe_travel.html]: miniView?

Comment (zarov) [reply examples/globe_travel.html]: Bad copy/paste

File: examples/3dtiles.html
Comment (gchoqueux) [inline examples/3dtiles.html:31]: alternative 1

```js 
var addLayerGui = menuGlobe.addLayerGUI.bind(menuGlobe);

itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
   var layer = new itowns.ColorLayer(config.id, config);
   view.addLayer(layer).then(addLayerGui);
});
``` 
alternative 2

```js 
itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
   view.addLayer(new itowns.ColorLayer(config.id, config)).then(addLayerGui);
});
``` 
alternative 3
```js 
function addLayerColorFromConfig(config) {
   view.addLayer(new itowns.ColorLayer(config.id, config)).then(addLayerGui);
}

itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerColorFromConfig);
``` 

I prefer Alternative 3

File: examples/globe.html
Comment (gchoqueux) [inline examples/globe.html]: Maybe do 2 layers, because visibility or opacity layer

File: examples/layersColorVisible.html
Comment (gchoqueux) [inline examples/layersColorVisible.html]: Could you write, its more comprehensive 
```js
itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
        var layer = new itowns.ColorLayer(config.id, config);
        layer.visible = true;
        view.addLayer(layer).then(menuGlobe.addLayerGUI.bind(menuGlobe));
});
```

File: examples/orientation_utils_planar.html
Comment (gchoqueux) [inline examples/orientation_utils_planar.html]: with the `return`

File: src/Core/Prefab/GlobeView.js
Comment (gchoqueux) [inline src/Core/Prefab/GlobeView.js]: is in other PR #904 ?

File: src/Layer/ColorLayer.js
Comment (gchoqueux) [inline src/Layer/ColorLayer.js:50]: why you remove example?

File: docs/tutorials/Create-a-simple-globe.md
Comment (gchoqueux) [inline docs/tutorials/Create-a-simple-globe.md:1]: :+1:

**Review by gchoqueux — CHANGES_REQUESTED** [review]

File: src/Layer/ColorLayer.js
Comment (zarov) [reply src/Layer/ColorLayer.js:50]: Because this type of adding a layer is going to be deprecated (if I understand correctly), so no need for this example to stay.

File: src/Core/Prefab/GlobeView.js
Comment (zarov) [reply src/Core/Prefab/GlobeView.js]: Yes, this is the same commit, I'm waiting for #904 to be merged before rebasing this one.

File: examples/orientation_utils_planar.html
Comment (zarov) [reply examples/orientation_utils_planar.html]: Because `addLayerCb` (which I forgot to remove) returned it, but there is no need indeed.

File: examples/layersColorVisible.html
Comment (zarov) [reply examples/layersColorVisible.html]: Ok

File: examples/globe.html
Comment (zarov) [reply examples/globe.html]: Ok, but we could have an other example that shows on layer in multiple view

File: examples/3dtiles.html
Comment (zarov) [reply examples/3dtiles.html:31]: There is only one `ColorLayer` to add, I don't see the point of doing that ?

Comment (gchoqueux) [reply examples/3dtiles.html:31]: it's a remark about the whole PR

File: examples/globe.html
Comment (gchoqueux) [reply examples/globe.html]: yes

File: src/Layer/ColorLayer.js
Comment (gchoqueux) [reply src/Layer/ColorLayer.js:50]: the example could be rewritten?

File: examples/3dtiles.html
Comment (zarov) [reply examples/3dtiles.html:31]: Ok, I rewrite all examples that have multiple layers of the same type added from a configuration file.

**Comment by zarov** [comment]
Rebased after merging of #904

File: src/Core/Prefab/GlobeView.js
Comment (gchoqueux) [reply src/Core/Prefab/GlobeView.js]: Ok don't forget to drop this commit when you rebase

File: src/Layer/Layer.js
Comment (gchoqueux) [inline src/Layer/Layer.js:77]: Have you add breaking change on your message commit?

Comment (zarov) [reply src/Layer/Layer.js:77]: No breaking change yet as there is a deprecation warning message and we still support the current way of adding layers.

Comment (zarov) [reply src/Layer/Layer.js:77]: Oh sorry I misunderstood, yes I'll add a breaking change in my message.

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**
